client/my-sites/controller.js

```diff
@@ -145,26 +145,26 @@ function isPathAllowedForDomainOnlySite( pathname, domainName ) {
 
 function onSelectedSiteAvailable( context ) {
 	const selectedSite = sites.getSelectedSite();
-	const state = context.store.getState();
+	const getState = () => context.store.getState();
```

Comment:
`getState()` returns a different object after a state transition after `dispatch` that happens later in this method.

If you meant that you can call each time `context.store.getState()`, that is right, but it's long, i prefer just write `getState()` if you don't mind =)